redhat-cop/containers-quickstarts
# Update ipa quickstart for OCP4

#### What is this PR About?
In order to run the ipa quickstart on OCP4, a certain sebool needs to be set. This PR does some cleanup on this quickstart and adds a machineconfig that puts this sebool in place.

#### How do we test this?
On OCP4 -- run the following:
```
ansible-playbook -i .applier/ roles/openshift-applier/playbooks/openshift-cluster-seed.yml -e 'include_tags=ipa_machineconfig'
```
You can then confirm that the machineconfig has been put in place by monitoring your nodes and machineconfigpools with

```
oc get nodes
oc get machineconfigpools
```
When looking at the machineconfigpools you can see that you are ready to proceed when the worker pool indicates that it has been updated and that updating status is set to false (i.e. no longer updating). For your nodes, you'll then see that your worker nodes are ready and schedulable again.

Once this is in place, you can then run the rest of the applier as expected:

```
ansible-playbook -i .applier/ roles/openshift-applier/playbooks/openshift-cluster-seed.yml -e 'exclude_tags=ipa_machineconfig'
```



cc: @redhat-cop/day-in-the-life
